servers/rendering/shader_language.cpp

```diff
@@ -4689,8 +4689,8 @@ Variant ShaderLanguage::get_default_datatype_value(DataType p_type, int p_array_
 				}
 				value = Variant(array);
 			} else {
-				VariantInitializer<float>::init(&value);
-				VariantDefaultInitializer<float>::init(&value);
+				VariantInitializer<double>::init(&value);
+				VariantDefaultInitializer<double>::init(&value);
```

Comment:
I think the shader language always uses 32-bit floats, although, in this case I guess we're initializing a `Variant` which will use a `double` internally? So, perhaps this is right